engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigratorImpl.java

```diff
@@ -126,6 +132,18 @@ private boolean handleMigrationTask(
     }
   }
 
+  private void logMigrationTaskAlreadyExecuted(
+      final MigrationTask migrationTask, final int index, final int total) {
+    LOGGER.info(
+        "Migration was executed before "
+            + migrationTask.getIdentifier()
+            + " migration ("
+            + index
+            + "/"
+            + total
```

Comment:
🔧 Instead of a separate method, you could extract a string template as constant and log inline